octobot_trading/exchanges/parser/orders_parser.py

```diff
@@ -0,0 +1,730 @@
+import decimal
+
+from octobot_trading import constants
+from octobot_trading.enums import (
+    ExchangeConstantsFeesColumns,
+    ExchangeConstantsOrderColumns as OrderCols,
+    OrderStatus,
+    TradeOrderSide,
+    TraderOrderType,
+)
+from octobot_trading.enums import ExchangeConstantsMarketPropertyColumns as MarketCols
+from octobot_trading.enums import TradeOrderType as TradeOrderType
+import octobot_trading.exchanges.parser.util as parser_util
+
+
+class OrdersParser(parser_util.Parser):
+    """
+    use OrdersParser as a base class if you implement
+    a new parser for a non ccxt or crypto feed exchange
+
+    parser usage:   parser = OrdersParser(exchange)
+                    orders = await parser.parse_orders(raw_orders)
+                    order = await parser.parse_order(raw_order)
+    """
+
+    TEST_AND_FIX_SPOT_QUANTITIES: bool = False
+    TEST_AND_FIX_FUTURES_QUANTITIES: bool = False
```

Comment:
Why is it optional? (and disabled by default)

Reply:
While we need it for bybit spot, there is also a high chance that it might cause issues with other exchanges.
As its using cost and remaining as a basis to recompute the other values